test/unit/gateways/vantiv_express_test.rb

```diff
@@ -0,0 +1,490 @@
+require 'test_helper'
+
+class VantivExpressTest < Test::Unit::TestCase
+  include CommStub
+
+  def setup
+    @gateway = VantivExpressGateway.new(account_id: '', account_token: '', application_id: '', acceptor_id: '', application_name: '', application_version: '')
```

Comment:
Should we be using the values in the `test/fixtures.yml` file instead of blank values?